### DISCUSSION TITLE
Auth0 Organisation Invitiations

### DISCUSSION THREAD

**Discussion by ollie-bud** [sergiodxa/remix-auth#269] [Q&A]
Auth0 supports invitations - where I can add a user to a organisation - they get an email and they can set their password themselves.

<img width="1122" alt="image" src="https://github.com/sergiodxa/remix-auth/assets/119597653/4183be3d-61e5-4bc1-8053-c7a5df43a21e">

The invitation url looks like `https://0.0.0.0:8788/?invitation={foo}&organization={bar}&organization_name={baz}`.

I'm not _quite_ sure how to implement this with `remix-auth-auth0` however, and scratching my head a little. Is this a thing that's supported? 

Thanks so much, Ollie

**Comment by sergiodxa** [comment]
This should be discussed in the Auth0Strategy repository as it's specific of Auth0 and not related to Remix Auth core.

But also I doubt this Auth0 feature is part of OAuth2 or OIDC flows. I think it you should just create a route in your Remix app, grab those search params and hit Auth0 API to accept the invite.

**Comment by ollie-bud** [comment]
Ah sorry for the wrong place! I just followed the github create question flow and it ended up here - but I might have got lost along the way.
I'll have a look into the Auth0 docs a bit more - and thank you so much for the incredibly prompt response (and the amazing work you do for the remix community) 🙏
